src/js/tech/flash.js

```diff
@@ -321,7 +321,7 @@ Flash['checkReady'] = function(tech){
 // Trigger events from the swf on the player
 Flash['onEvent'] = function(swfID, eventName){
   let player = Lib.el(swfID)['player'];
-  player.trigger(eventName);
+  player.tech.trigger(eventName);
```

Comment:
This is an out-dated diff (I think I misplaced my comment for the placeholder). It is now:
```javascript
let tech = Lib.el(swfID)['tech'];
tech.trigger(eventName);
```

and the tech is saved within the element instead of the player.